mmeds/util.py

```diff
@@ -1366,3 +1381,21 @@ def run_analysis(path, tool_type, testing=False):
         Logger.debug(e.output)
         Logger.debug(e)
         raise e
+
+
+def upload_study_directly(queue, study_name, subject, subject_type, specimen, user):
```

Comment:
I personally would use 'local' instead of 'directly' i.e. upload_local_study since you're uploading files that are on the same file system as the mmeds install (as contrasted with remote upload)